doc/help/elektra-data-structures.md

```diff
@@ -366,3 +366,46 @@ The algorithm uses the `syncbits` to decide if the key set needs to be
 synchronised.
 
 Continue reading [with the error handling](elektra-error-handling.md).
+
+## Order Preserving Minimal Perfect Hash Map
+
+All structs are defined [in](/src/include/opmphm.h).
+
+### Vstack
+
+The `Vstack` is a virtual stack implementation with dynamical memory allocation.
+The allocation doubles the space if the `Vstack` is full and reduces the space by the half
```

Comment:
> … and reduces the space by **the** half if the memory usage **gets under** a quarter.

Maybe better: and reduces the space by half if the memory usage **drops below** a quarter.